src/mediafileattribute.cpp

```diff
@@ -594,7 +594,6 @@ bool MediaProperties::isMediaFilenameExt(const std::string& ext)
         ".mka.mks.mlp.mov.mp1.mp1v.mp2.mp2v.mp3.mp4.mp4v.mpa1.mpa2.mpeg.mpg.mpgv.mpv.mqv.ogg.ogm.ogv"
         ".omg.opus.qt.sls.spx.thd.tmf.trp.ts.ty.vc1.vob.vr.w64.wav.webm.wma.wmv.";
 
-    assert(ext.size() >= 2 && ext[0] == '.');
```

Comment:
Alright, assert removed since there are filenames like `my file a...` which don't have an extension (it is `.`), so the assert fails (already failing in prod if the case happens).